### PR TITLE
Issue 18268 when es is down the form are not getting render

### PR DESCRIPTION
**Pull request by freddyucv**
When Elasticsearch is down we are getting a error trying to search by the Language variable here:

https://github.com/dotCMS/core/compare/issue-18268-When-ES-is-down-the-form-are-not-getting-render?expand=1#diff-2fbc504febcbcbd478a77db00d87e7ecR264

Fix: when ES is down the Field Internationalization is not going to work but we want that the form still could be rendered

### CODE REVIEW

File: dotCMS/src/main/java/com/dotcms/contenttype/model/field/layout/FieldLayoutColumnSerializer.java [hunk 1]
```diff
@@ -57,17 +54,24 @@ public void serialize(
...
 
+        setFieldInternationalization(serializerProvider, fieldsMap);
+
+        jsonGenerator.writeObjectField("fields", fieldsMap);
+
+        jsonGenerator.writeEndObject();
+        jsonGenerator.flush();
+    }
+
+    private void setFieldInternationalization(SerializerProvider serializerProvider, List<Map<String, Object>> fieldsMap) {
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/model/field/layout/FieldLayoutColumnSerializer.java:65]: Codacy Issue found: [Parameter 'serializerProvider' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)

File: dotCMS/src/main/java/com/dotcms/contenttype/model/field/layout/FieldUtil.java [hunk 2]
```diff
@@ -4,13 +4,16 @@
 import com.dotcms.contenttype.model.type.ContentType;
 import com.dotcms.contenttype.transform.contenttype.ContentTypeInternationalization;
 import com.dotcms.contenttype.transform.field.JsonFieldTransformer;
+import com.dotcms.exception.ExceptionUtil;
 import com.dotmarketing.business.APILocator;
 import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.portlets.fileassets.business.FileAssetAPIImpl;
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/model/field/layout/FieldUtil.java:10]: Codacy Issue found: [Avoid unused imports such as 'com.dotmarketing.portlets.fileassets.business.FileAssetAPIImpl'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)

File: dotCMS/src/main/java/com/dotcms/contenttype/model/field/layout/FieldUtil.java [hunk 3]
```diff
@@ -245,14 +257,22 @@ public static void setFieldInternationalization(
...
+                final String key = String.format("%s.%s.%s", contentType.variable(), fieldMap.get("variable"), propertyName);
+                final String i18nValue = APILocator.getLanguageVariableAPI().getLanguageVariable(
+                        key, languageId, user, live, user == null);
+
+                if (!i18nValue.equals(key) && !i18nValue.equals(fieldMap.get(propertyName).toString())) {
+                    fieldMap.put(propertyName, i18nValue);
+                }
+            }
+        } catch (DotRuntimeException e) {
+            if ( ExceptionUtil.causedBy(e, ConnectException.class)) {
```
Comment (dsilvam) [inline dotCMS/src/main/java/com/dotcms/contenttype/model/field/layout/FieldUtil.java:272]: can we write a test for this?

File: dotCMS/src/main/java/com/dotcms/contenttype/model/field/layout/FieldUtil.java [hunk 4]
```diff
@@ -236,6 +239,15 @@ public static Field copyField(final Field field, final int newSortOrder) {
...
+    /**
+     * Apply Internationalization to field property using a Language Variable, for each field property set into
+     * fieldMap a Language Variable with the name contentTypeVariable.fieldVariable.propertyName is searched, if it
+     * exists then the field's property value is replaced by the language variable value
+     *
+     * @param contentType
+     * @param contentTypeInternationalization set the mode, language and user to search the Language Variable
+     * @param fieldMap field properties
+     */
     public static void setFieldInternationalization(
```
Comment (dsilvam) [inline dotCMS/src/main/java/com/dotcms/contenttype/model/field/layout/FieldUtil.java:251]: This method sounds like a good candidate to be moved to an API. Too much for a util method?

**Review by erickgonzalez — APPROVED** [review]

**Review by jdotcms — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/contenttype/model/field/layout/FieldUtil.java (hunk 4 above)
Comment (freddyucv) [reply dotCMS/src/main/java/com/dotcms/contenttype/model/field/layout/FieldUtil.java]: done https://github.com/dotCMS/core/pull/18281/commits/c5d753c9f171f339c058cfc4df2a6d8a16a5725a

File: dotCMS/src/main/java/com/dotcms/contenttype/model/field/layout/FieldUtil.java (hunk 3 above)
Comment (freddyucv) [reply dotCMS/src/main/java/com/dotcms/contenttype/model/field/layout/FieldUtil.java]: done https://github.com/dotCMS/core/pull/18281/commits/fc44d05d8a9e9041ebf7df076ce9de4b45694bc7#diff-174105054591969073d7cc9d3406b5d5R27

**Review by dsilvam — APPROVED** [review]

File: dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+        final boolean live = true;
+        final User user = mock(User.class);
+        final Object object = mock(Object.class);
+
+        final ContentType contentType = mock(ContentType.class);
+        final ContentTypeInternationalization contentTypeInternationalization = mock(ContentTypeInternationalization.class);
+        final Map<String, Object> fieldMap = map(
+                "attribute_1", "value_1",
+                "attribute_2", "value_2",
+                "attribute_3", "value_3",
```
Comment (dev-dotcms) [inline dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java:72]: Codacy Issue found: [The String literal "value_3" appears 4 times in this file; the first occurrence is on line 72](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)

File: dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+                "variable", "testField",
```
Comment (dev-dotcms) [inline dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java:73]: Codacy Issue found: [The String literal "variable" appears 4 times in this file; the first occurrence is on line 73](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)

File: dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+        final long languageId = 1;
+        final boolean live = true;
+        final User user = mock(User.class);
+        final Object object = mock(Object.class);
+
+        final ContentType contentType = mock(ContentType.class);
+        final ContentTypeInternationalization contentTypeInternationalization = mock(ContentTypeInternationalization.class);
+        final Map<String, Object> fieldMap = map(
+                "attribute_1", "value_1",
+                "attribute_2", "value_2",
```
Comment (dev-dotcms) [inline dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java:71]: Codacy Issue found: [The String literal "attribute_2" appears 4 times in this file; the first occurrence is on line 71](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)

File: dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+import com.dotmarketing.portlets.contentlet.business.ContentletAPI;
+import com.liferay.portal.model.User;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.net.ConnectException;
+import java.util.Map;
+
+import static com.dotcms.util.CollectionsUtils.map;
+import static org.mockito.Matchers.*;
```
Comment (dev-dotcms) [inline dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java:22]: Codacy Issue found: [Avoid unused imports such as 'org.mockito.Matchers'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/FieldAPI.java [hunk 9]
```diff
@@ -229,4 +220,37 @@ default FieldAPI instance() {
...
+	 * Apply Internationalization to field property using a Language Variable, for each field property set into
+	 * fieldMap a Language Variable with the name contentTypeVariable.fieldVariable.propertyName is searched, if it
+	 * exists then the field's property value is replaced by the language variable value
+	 *
+	 * @param contentType
+	 * @param contentTypeInternationalization set the mode, language and user to search the Language Variable
+	 * @param fieldMap field properties
+	 * @return  The Field's properties with the new internationalization values
+	 */
+	public Map<String, Object> getFieldInternationalization(
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/FieldAPI.java:250]: Codacy Issue found: [Unnecessary modifier 'public' on method 'getFieldInternationalization': the method is declared in an interface type](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)

File: dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java [hunk 10]
```diff
@@ -0,0 +1,160 @@
+package com.dotcms.contenttype.business;
+
+import com.dotcms.contenttype.business.FieldAPIImpl;
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotcms.contenttype.transform.contenttype.ContentTypeInternationalization;
+import com.dotcms.languagevariable.business.LanguageVariableAPI;
+import com.dotcms.system.event.local.business.LocalSystemEventsAPI;
+import com.dotmarketing.business.APILocator;
```
Comment (dev-dotcms) [inline dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java:8]: Codacy Issue found: [Avoid unused imports such as 'com.dotmarketing.business.APILocator'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)

File: dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java [hunk 11]
```diff
@@ -0,0 +1,160 @@
...
+    public void shouldFieldInternationalizationWork(){
+        final long languageId = 1;
+        final boolean live = true;
+        final User user = mock(User.class);
+        final Object object = mock(Object.class);
+
+        final ContentType contentType = mock(ContentType.class);
+        final ContentTypeInternationalization contentTypeInternationalization = mock(ContentTypeInternationalization.class);
+        final Map<String, Object> fieldMap = map(
+                "attribute_1", "value_1",
```
Comment (dev-dotcms) [inline dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java:70]: Codacy Issue found: [The String literal "attribute_1" appears 4 times in this file; the first occurrence is on line 70](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)

File: dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java [hunk 12]
```diff
@@ -0,0 +1,160 @@
+package com.dotcms.contenttype.business;
+
+import com.dotcms.contenttype.business.FieldAPIImpl;
```
Comment (dev-dotcms) [inline dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java:3]: Codacy Issue found: [No need to import a type that lives in the same package](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)

**Review by dsilvam — APPROVED** [review]

File: dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java (hunk 5 above)
Comment (dev-dotcms) [inline dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java:72]: Codacy Issue found: [The String literal "value_3" appears 4 times in this file; the first occurrence is on line 72](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)

File: dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java (hunk 11 above)
Comment (dev-dotcms) [inline dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java:70]: Codacy Issue found: [The String literal "attribute_1" appears 4 times in this file; the first occurrence is on line 70](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)

File: dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java (hunk 7 above)
Comment (dev-dotcms) [inline dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java:71]: Codacy Issue found: [The String literal "attribute_2" appears 4 times in this file; the first occurrence is on line 71](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)

File: dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java (hunk 6 above)
Comment (dev-dotcms) [inline dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java:73]: Codacy Issue found: [The String literal "variable" appears 4 times in this file; the first occurrence is on line 73](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)

File: dotCMS/src/main/java/com/dotcms/contenttype/business/FieldAPI.java (hunk 9 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/contenttype/business/FieldAPI.java:250]: Codacy Issue found: [Unnecessary modifier 'public' on method 'getFieldInternationalization': the method is declared in an interface type](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)

File: dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java (hunk 12 above)
Comment (dev-dotcms) [inline dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java:3]: Codacy Issue found: [No need to import a type that lives in the same package](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)

File: dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java (hunk 8 above)
Comment (dev-dotcms) [inline dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java:22]: Codacy Issue found: [Avoid unused imports such as 'org.mockito.Matchers'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)

File: dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java (hunk 10 above)
Comment (dev-dotcms) [inline dotCMS/src/test/java/com/dotcms/contenttype/business/FieldAPIImplTest.java:8]: Codacy Issue found: [Avoid unused imports such as 'com.dotmarketing.business.APILocator'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)

**Pull request merged**